### ISSUE TITLE
ar_conurbano_buenos_aires_grid

### ISSUE THREAD

**Issue by SantiagoEsteban** [hafen/geofacet#282]
Grid data:

```
name,code,row,col
malvinas_argentinas,1,2,4
tigre,2,2,5
san_fernando,3,2,6
jose_c_paz,4,2,3
san_isidro,5,3,6
san_miguel,6,3,5
vicente_lopez,7,4,7
moreno,8,4,3
general_san_martin,9,4,6
hurlingham,10,4,5
ituzaingo,11,4,4
caba,12,5,7
moron,14,5,5
tres_de_febrero,15,5,6
merlo,13,6,5
avellaneda,16,6,8
lanus,17,6,7
la_matanza,18,6,6
quilmes,19,7,8
esteban_echeverria,20,7,6
ezeiza,21,7,5
lomas_de_zamora,22,7,7
berazategui,23,8,9
almirante_brown,24,8,7
florencio_varela,25,8,8
```

![map](https://es.wikipedia.org/wiki/Gran_Buenos_Aires#/media/Archivo:Mapa_de_la_Gran_Buenos_Aires.svg)

**Comment by stedy** [comment]
Hi this looks great but is it complete? There are many other Partidos listed here that were not on your grid: https://en.wikipedia.org/wiki/Partidos_of_Buenos_Aires

**Comment by SantiagoEsteban** [comment]
Yes, actualy I wanted to represent the greater Buenos Aires area which includes the autonomous city of buenos aires + some department/partidos of the Buenos Aires Province, but not all of them. 

https://en.wikipedia.org/wiki/Greater_Buenos_Aires

Here is the revised version include all three ringgs of the greater buenos aires area.

name,code,row,col
escobar,26,1,3
pilar,26,1,2
jose_c_paz,4,2,2
malvinas_argentinas,1,2,3
tigre,2,2,4
san_fernando,3,2,5
san_miguel,6,3,4
san_isidro,5,3,5
moreno,8,4,2
ituzaingo,11,4,3
hurlingham,10,4,4
general_san_martin,9,4,5
vicente_lopez,7,4,6
general_rodriguez,28,4,1
merlo,13,5,3
moron,14,5,4
tres_de_febrero,15,5,5
caba,12,5,6
marcos_paz,29,5,2
la_matanza,18,6,5
lanus,17,6,6
avellaneda,16,6,7
ezeiza,21,7,4
esteban_echeverria,20,7,5
lomas_de_zamora,22,7,6
quilmes,19,7,7
san_vicente,32,9,5
canuelas,30,8,4
almirante_brown,24,8,6
florencio_varela,25,8,7
berazategui,23,8,8
presidente_peron,31,8,5

**Comment by stedy** [comment]
![](https://upload.wikimedia.org/wikipedia/commons/0/0b/Great_buenos_aires.png)
